src/config/utils.ts

```diff
@@ -264,6 +264,10 @@ export const pairsToExchangeMapping: { [key: string]: string } = {
   SEIWBTC: 'BINANCE:SEIBTC',
   SEIBNB: 'BINANCE:SEIBNB',
   SEITUSD: 'BINANCE:SEITUSD',
+  HBARUSDC: 'BINANCE:HBARUSDC',
+  HBARUSDT: 'BINANCE:HBARUSDT',
+  HBARBTC: 'BINANCE:HBARBTC',
+  HBARBNB: 'BINANCE:HBARBNB',
```

Comment:
Relates to TradingView integration within the application. In this context, it maps the key HBARUSDC to the TradingView symbol BINANCE:HBARUSDC. This mapping allows the application to use the correct identifier when displaying or fetching the HBAR/USDC pair's chart data from TradingView
Note that: Must enable priceChart is `tradingView`, so that the app will get chart data from TradingView